Specs/Core/ColorSpec.js

```diff
@@ -358,6 +358,17 @@ describe("Core/Color", function () {
     expect(c).toEqual(Color.LIME);
   });
 
+  it("fromCssColorString understands the color string even with any number of unnecessary leading, trailing or middle spaces", function () {
+    expect(Color.fromCssColorString(" rgb( 108, 136, 255)")).toBeDefined();
+    expect(Color.fromCssColorString("rgb( 108, 136, 255) ")).toBeDefined();
+    expect(Color.fromCssColorString("  #FF0000")).toBeDefined();
+    expect(Color.fromCssColorString("#FF0  ")).toBeDefined();
+    expect(
+      Color.fromCssColorString(" hsla(720,   100%, 50%, 1.0)  ")
+    ).toBeDefined();
+    expect(Color.fromCssColorString("hsl (720, 100%, 50%)")).toBeDefined();
```

Comment:
For a better test, can you update these to be `.toEqual(new Color(...))` instead of `toBeDefined()`?